react/src/components/organisms/TabContainer/tab.js

```diff
@@ -31,15 +35,53 @@ class Tab extends React.Component {
             this.defaultSet = true;
           }
           return(
-            <button
-              className={tabClasses}
-              onClick={(e) => {
-                this.scrollIntoView(e.target);
-                setActiveTab(this.tabIdent, this.props.children)
-              }}
-            >
-            {this.props.title}
-            </button>
+            <li role="presentation" className={tabClasses}>
+              <button
+                role="tab"
+                id={this.tabIdent}
+                onClick={(e) => {
+                  this.scrollIntoView(e.target);
+                  setActiveTab(this.tabIdent, this.props.children)
+                }}
+                aria-selected={active}
+                tabIndex={!active ? -1 : 0}
+                aria-describedby={this.props.a11yID}
+
+
+    // THIS EVENT IS ONLY AVAILABLE TO THE ONFOCUS TAB.
+                onKeyDown = {(e) => {
+                  if (e.keyCode === 37 || e.keyCode === 39) {
+                    e.preventDefault();
+                    e.stopPropagation();
+
+                    if (this.tabIdent !== document.activeElement.id) {
+                      console.log('Set the active tab!');
+                      setActiveTab(document.activeElement.id, this.props.children);
```

Comment:
this.tabIdent is already set to the same value of document.activeElement.id:
```suggestion
                      setActiveTab(this.tabIdent, this.props.children);
```

Reply:
Changed to `this.props.tabIdent`.